docs/pages/access-controls/guides/headless-login.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: Headless Login (Preview)
+description: Headless Login
+---
+
+## Introduction
+
+<Admonition type="warning" title="Preview">
+  Headless login is currently in Preview mode.
+</Admonition>
+
+Headless login provides Teleport users with a secure way to authenticate with WebAuthn from a remote machine. This enables use cases which usually are not compatible with WebAuthn features, such as:
+
+- Logging into Teleport with WebAuthn from a remote dev box
+- Connecting to a Teleport Node from a remote dev box with Per-session MFA 
+- Performing `tsh scp` from one Teleport Node to another with Per-session MFA
+- Logging into Teleport on a machine without a WebAuthn compatible browser
+
+<Admonition type="note" title="Headless Login Support">
+  Headless login only supports following `tsh` commands:
+  
+  - `tsh ls`
+  - `tsh ssh`
+  - `tsh scp`
+
+  In the future, headless login may be extended to other `tsh` commands.
+</Admonition>
+
+## Prerequisites
+
+- A Teleport cluster with WebAuthn configured.
+  See the [Second Factor: WebAuthn](./webauthn.mdx) guide.
+- WebAuthn hardware device, such as YubiKey or SoloKey.
+- A Web browser with [WebAuthn support](
+  https://developers.yubico.com/WebAuthn/WebAuthn_Browser_Support/).
+
+A Teleport cluster capable of WebAuthn is automatically capable of Headless login.
```

Comment:
I'm not sure if I understand it. First, your cluster needs to be v12.2, so not any cluster is capable.

Reply:
The point is that Headless will automatically work if WebAuthn is configured. I can add the v12.2 caveot.